gulpfile.js

```diff
@@ -362,16 +359,17 @@ gulp.task('karma:coverage', function(done) {
 // Drops the MongoDB database, used in e2e testing
 gulp.task('dropdb', function (done) {
   // Use mongoose configuration
-  var mongoose = require('./config/lib/mongoose.js');
+  var mongooseService = require('./config/lib/mongoose');
```

Comment:
Same as earlier:

Is there any reason why this `require` is inside the task and not on top of the gulp file with all the other dependencies?